### ISSUE TITLE
Great component!

### ISSUE THREAD

**Issue by ajp8164** [PedroBern/react-native-collapsible-tab-view#382]
People don't seem to say thank you to devs too much - thank you for this component!

**Comment by VictorioMolina** [comment]
I totally agree. The only problem I have found is not being able to scroll on header touchables. But not a really serious problem imo. Currently, this library is better than react-native-tab-view.
